packages/popover/src/index.tsx

```diff
@@ -195,6 +200,7 @@ export const Popover: React.FC<PopoverProps> = ({
   onClose = () => {},
   onOpenComplete = () => {},
   onCloseComplete = () => {},
+  onOpenStarted = () => {},
```

Comment:
There is `onClose` which is called on closing animation start so this should be named `onOpen`